spec/scorecardSpec.js

```diff
@@ -0,0 +1,35 @@
+describe('Scorecard', function() {
+  var scorecard;
+  beforeEach(function() {
+    scorecard = new Scorecard();
+  });
+  // The first test - Does my frame contain 3 objects [Score1, Score2, Score3[sum]]
+
+  it('It allows firstbowl value to be added to be added to frame', function() {
+    expect(scorecard.firstBowl(5)).toEqual(scorecard.bowlOne)
```

Comment:
nice test, I like it!

Reply:
I'm not sure what this is testing. What is the "behaviour" of firstBowl? From a user of your class point of view, what is it supposed to produce?
The confusion here for me is because you have code on both sides of the matcher (`scorecard.firstBowl(5)` and `scorecard.bowlOne`) and it's not obvious to test readers what the values for each should be.
The expected output should be fixed, instead.
Let me know if that's not clear!